backend/authentication/password_validators.py

```diff
@@ -24,3 +25,15 @@ def get_help_text(self):
             "Your password must contain at least 1 numeric character and 1 alphabetic "
             "character"
         )
+
+
+class PasswordStrengthValidator:
+    def validate(self, password, user=None, resultLabel, result):
+        result = zxcvbn(password) resultLabel = "Score: \(result?.score ?? 0)\n"
+        return resultLabel
```

Comment:
ok, so the idea here in this function is to raise a `django.core.exceptions.ValidationError` when the password doesn't pass validation. so in pseudo code:
```
score = zxcvbn(password)
if score is too low:
    raise the validation error with a helpful message of what's wrong and how to fix it
```
and that's it! the function doesn't need to return anything. it just needs to do a check and see if the password looks good. if it doesn't, it raises that exception.